### PR TITLE
JavaScript Objects

### PR DESCRIPTION
**Pull request by over-geek**
## Awesome Books Project: JavaScript Objects

In this milestone, we:
- Added a basic UI with plain HTML
- Created a collection that keeps a list of books (hint: you can use an array of objects for that).
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection 
- Displayed all books saved in the collection at the top part of the page.

Collaborators: 
- Agyare Kissi Kenneth @over-geek 
- Ali Baba @Alibaba2023

### CODE REVIEW

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @over-geek and @Alibaba2023 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that does not have a readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+addBtn.addEventListener('click', addNewBook);
+
+document.addEventListener('click', (e) => {
+  if (e.target.className === 'remove-btn') {
+    Books.splice(e.target.id, 1);
+    e.target.parentNode.remove();
+    saveData();
+  }
```
Comment (Shedrack-Sunday) [inline script.js]: - [ ] Kindly make this  a standalone function. This make it reusable and organises your code better.
- [ ] The solution you provided seems to always keep the first index when deleted. i tried adding `5` books and deleted down to the `3rd` book, but it remained even on page reload. i'll advice you use a `filter` method to get the specifics book to be deleted
- Possible solution:
```
function removeBook(e) {
  if (e.target.className === 'remove-btn') {
    Books = Books.filter((book, index) => index != e.target.id);
    e.target.parentNode.remove();
    saveData();
  }
}

document.addEventListener('click', removeBook);
```

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @over-geek     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done  extremely well so far in this project,  However, there's one adjustment  you need to make  to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   

  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you to make In line with the requirements 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: script.js (hunk 1 above)
Comment (over-geek) [reply script.js]: thank you @Shedrack-Sunday for your feedback

**Review by amon-cofie — APPROVED** [review]
Hi @over-geek and @Alibaba2023 , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have a descriptive pull request summary
🚀 You implemented the requested changes very well 
🚀 You used GitHub flow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**